.storybook/custom/components/Header.js

```diff
@@ -1,13 +1,17 @@
 import packageJson from '../../../package.json';
 import React from 'react';
 
-const Header = ({onThemeChange}) => {
+const Header = ({onThemeChange, onDirectionalityChange}) => {
     return (
         <header className='fddocs-header'>
             <section className='fddocs-header--left'>
                 <span>Fundamental Styles</span>
             </section>
             <section className='fddocs-header--right'>
+                <select className='fddocs-select' onChange={(e) => onDirectionalityChange(e)}>
+                    <option value='ltr'>Left-to-Right</option>
+                    <option value='rtl'>Right-to-Left</option>
```

Comment:
Should these just be `LTR` and `RTL` or do we need to explicitly call out "Left-to-Right"? Maybe @jessmetivier knows the best way to communicate this

Reply:
@jbadan either way - whatever @jessmetivier say :)

Reply:
Since this is only a label and our users are both developers and designers, it needs to be explicitly stated.